nipoppy/workflows/base.py

```diff
@@ -90,7 +90,7 @@ def generate_fpath_log(
 
     def log_command(self, command: str):
         """Write a command to the log with a special prefix."""
-        self.logger.info(f"{self.log_prefix_run} {command}")
+        self.logger.info(f"{self.log_prefix_run} {command}", extra={"markup": False})
```

Comment:
A reason for `extra` arguments would be appreciated by the other fellow mortal devs.